src/DynamoCore/DynamoCore.csproj

```diff
@@ -102,6 +103,7 @@ limitations under the License.
     </Reference>
     <Reference Include="System.Xml" />
     <Reference Include="System.Xml.Linq" />
+    <Reference Include="WindowsBase" />
```

Comment:
I think `WindowsBase` is also platform specific, if that's right, then it can't be included here.